cms/io/triggeredservice.py

```diff
@@ -35,7 +35,7 @@
 logger = logging.getLogger(__name__)
 
 
-class Executor(object, metaclass=ABCMeta):  # pylint: disable=R0921
+class Executor(metaclass=ABCMeta):  # pylint: disable=R0921
```

Comment:
I think you can remove the R0921 thingie, now that you're using metaclass.

Reply:
Done. (The pylint warning was because we were defining an abstract class without immediately using it, however it seems that warning is gone, so the disabling can be removed)